pallets/identity/rpc/runtime-api/src/lib.rs

```diff
@@ -41,6 +41,18 @@ pub enum DidStatus {
     Exists,
     CddVerified,
 }
+#[derive(Eq, PartialEq, Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+pub enum AuthorizationType {
```

Comment:
Instead of defining this here, can we please define this alongside the `AuthorizationData` in primitives?

Reply:
Alright no problem